base/message/schema/self-issued-id-token-schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Self-Issued ID Token",
+    "type": "object",
+    "properties": {
+        "sub": {
+            "type": "string",
+            "description": "Subject identifier"
+        },
+        "iss": {
+            "type": "string",
+            "description": "Issuer identifier"
+        },
+        "iat": {
+            "type": "integer",
+            "description": "Issued at time (seconds since epoch)"
+        },
+        "exp": {
+            "type": "integer",
+            "description": "Expiration time (seconds since epoch)"
+        },
+        "jti": {
+            "type": "string",
+            "description": "JWT ID"
+        },
+        "bearer_access_scope": {
+            "type": "string",
+            "description": "Authorization request parameter containing a list of space-delimited scopes for the token claim"
+        },
```

Comment:
```suggestion
```
see above

Reply:
As per our last discussion, I believe it was decided that a "token" will be present only if "bearer_access_scope" is present. So these changes are reflecting that condition.

Reply:
Removed bearer_access_scope